carbondoomsday/settings.py

```diff
@@ -141,6 +141,13 @@ class CeleryAndCeleryBeat():
             ),
             'schedule': timedelta(minutes=50)
         },
+        'update-email-signature': {
+            'task': (
+                'carbondoomsday.email_signature.tasks.'
+                'update_email_signature'
+            ),
+            'schedule': timedelta(days=1)
```

Comment:
Perhaps this should be fired manually from the other celery tasks. For example, if the latest data updates at some strange time and this runs before that, the signature will not have the latest data. I think you'll want to try with `apply_async`.